src/supports.js

```diff
@@ -2,20 +2,19 @@
 /**
  * Setup dummy elements
  */
-import { prefixes, domPrefixes } from '../supports/shared.js';
+import { prefixes, domPrefixes } from '../supports/src/data.js';
 import { IS_DEV } from './util.js';
 
-import property from '../supports/property.js';
-import value from '../supports/value.js';
-import descriptor from '../supports/descriptor.js';
-import descriptorvalue from '../supports/descriptorvalue.js';
-import selector from '../supports/selector.js';
-import atrule from '../supports/atrule.js';
-import mq from '../supports/mq.js';
-import variable from '../supports/variable.js';
-import Global from '../supports/global.js';
-import member from '../supports/member.js';
-import testExtends from '../supports/extends.js';
+import property from '../supports/src/types/css/property.js';
+import value from '../supports/src/types/css/value.js';
+import descriptor from '../supports/src/types/css/descriptor.js';
+import descriptorvalue from '../supports/src/types/css/descriptor-value.js';
+import selector from '../supports/src/types/css/selector.js';
+import atrule from '../supports/src/types/css/atrule.js';
+import mq from '../supports/src/types/css/mq.js';
+import Global from '../supports/src/types/js/global.js';
+import member from '../supports/src/types/js/member.js';
+import testExtends from '../supports/src/types/js/extends.js';
```

Comment:
I wonder if there's a way to do the same thing with this code and use `src/index.js` here too? How would we transform the `Supports` constant in that case? 🤔

Reply:
You don’t need the supports constant here, just use `import * as supports`

Reply:
I tried it but and it began to throw a error I don't understand:

<img width="522" height="152" alt="image" src="https://github.com/user-attachments/assets/b7f61d91-c248-439c-ad4d-386b749a5184" />

Reply:
Looks like some of our code expects an object similar to the `Supports` constant. 🤷‍♂️

This code produces the error:

<img width="470" height="295" alt="image" src="https://github.com/user-attachments/assets/7968661f-b162-46fa-9f85-fba6af8e05c4" />

Reply:
I definitely miss something...

Reply:
And I think we didn't export `extends`. 🤔

Yup, we didn't: https://github.com/browserscore/supports/blob/main/src/types/js/index-fn.js
It's intentional?

Reply:
No, it's not intentional

Reply:
[Fixed it](https://github.com/browserscore/supports/commit/eb0b1c22c5e0d4304b32c4fe3e70fae5e45ce0f8)